From 9e31d936bb10c6d1dddb16b40d7c904de3dd8d20 Mon Sep 17 00:00:00 2001
From: Nico Weber <thakis@chromium.org>
Date: Fri, 14 May 2021 19:42:33 +0000
Subject: [PATCH] build: Convert services/metrics/public/cpp to py3

Seems to just work.

Bug: 1205626
Change-Id: I581e509f5dd2333a2c8884b5de61a5c8bc11f40a
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2896055
Auto-Submit: Nico Weber <thakis@chromium.org>
Commit-Queue: Weilun Shi <sweilun@chromium.org>
Reviewed-by: Weilun Shi <sweilun@chromium.org>
Cr-Commit-Position: refs/heads/master@{#883083}
---
 src/services/metrics/public/cpp/BUILD.gn | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/services/metrics/public/cpp/BUILD.gn b/src/services/metrics/public/cpp/BUILD.gn
index e1ca96d4a2..e4e21acf3c 100644
--- a/src/services/metrics/public/cpp/BUILD.gn
+++ b/src/services/metrics/public/cpp/BUILD.gn
@@ -2,7 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//build/config/python.gni")
 import("//mojo/public/tools/bindings/mojom.gni")
 
 component("metrics_cpp") {
@@ -57,8 +56,7 @@ source_set("tests") {
   ]
 }
 
-# TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
-python2_action("gen_ukm_builders") {
+action("gen_ukm_builders") {
   script = "//tools/metrics/ukm/gen_builders.py"
 
   # Re-generate the outputs if the codegen code changes:
